EELS/EELS_elemental_mapping.ipynb

```diff
@@ -0,0 +1,789 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Creating and plotting elemental EELS maps\n",
+    "\n",
+    "In this tutorial, elemental maps of copper and zinc will be created from electron energy loss spectra (EELS). The data is recorded using a sample of copper and zinc oxide deposited on carbon nanotubes. The particles are likely to be very small, and the carbon is not completely covered. In this sample, the ratio of Zn and Cu is 3:1, and approximately 80 wt% of the sample is carbon. Due to the small amount of copper and zinc, the signal strength is low. To improve the signal strength without lowering the spatial resolution, the spectra were recorded with a high energy dispersion, resulting in low energy resolution. This means that any fine structure is not visible.\n",
+    "\n",
+    "For more information about the material system and how the results of this processing can be used, see the paper: https://doi.org/10.1016/j.cattod.2019.02.045\n",
+    "\n",
+    "This notebook requires `HyperSpy` 2.0 or higher. It also requires `eXSpy` which is a library which contains all of HyperSpy's EELS and EDX functionality.\n",
+    "\n",
+    "The main objective is to show how the functionality of HyperSpy can be used to create relative elemental maps from EEL spectra.\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "jp-MarkdownHeadingCollapsed": true
+   },
+   "source": [
+    "### Changes\n",
+    "\n",
+    "* 2017/09/27: Initial version by Ida Hjorth\n",
+    "* 2019/11/14: Update to HyperSpy 1.5, and minor improvements to text by Magnus Nord\n",
+    "* 2024/3/16: Update to work with HyperSpy 2.0, by Magnus Nord"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Table of contents"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "1. <a href='#load'> Load data</a> \n",
+    "2. <a href='#fitting'>Fitting Cu and Zn Hartree-Slater cross sections</a>\n",
+    "3. <a href='#save_restore'>Saving and restoring the model</a>\n",
+    "4. <a href='#plotting'>Creating elemental maps from the model</a>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# <a id='load'></a>1. Load data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib widget\n",
+    "import numpy as np\n",
+    "import hyperspy.api as hs"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "s_eels = hs.load('datasets/CuZn_EELS_mapping_tutorial.hspy')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Making elemental maps for this map is a bit tricky, due to the fairly weak edges, and the overlap between the Cu and Zn edges."
```

Comment:
```suggestion
    "Because of the overlap of the Cu (xxx eV) and Zn edges (xxx eV), it is very difficult to estimate background for the Zn edge. An solution to this problem is use the fit the background and edges as this is shown in this notebook."
```

Reply:
done!